build.xml

```diff
@@ -11,6 +11,7 @@
   <property name="dir-lib" value="lib" />
   <property name="dir-bin" value="bin" />
   <property name="dir-dist" value="dist" />
+  <property name="dir-test-report" value="test-output" />
```

Comment:
Please follow the pattern here: `name="dir-*" value="*"`, e.g. change then name to `name="dir-test-output"`.